emissionsapi/web.py

```diff
@@ -19,6 +19,31 @@
 logger = logging.getLogger(__name__)
 
 
+def parse_date(key):
```

Comment:
What about using `def parse_date(*key):` instead? That way you could use something like `@parse_date('begin', 'end`)`?

Reply:
Great idea. I have done that.